tests/test_connect.py

```diff
@@ -306,6 +306,11 @@ def test_client_revision(self):
         with self.created_client(client_revision=54032) as client:
             client.execute('SELECT 1')
 
+    def test_client_with_no_cert_validation(self):
+        with self.created_client(port=9440,
```

Comment:
Let's:
1) add `secureport=9440` into `setup.cfg`;
2) add attribute `secure_port` to class `tests.testcase.BaseTestCase`;
3) use `self.secure_port` here instead of hardcoding.

Reply:
Fixed